staging/com.smartnews/manifest.tt

```diff
@@ -0,0 +1,37 @@
+class @com.smartnews
+{
+  import loader from @org.thingpedia.v2();
+  import config from @org.thingpedia.config.none();
+
+  list query top_articles(in opt counter: Number #[prompt="How many news articles do you want?"],
+    			  out news_id: String,
+                          out title: String,
+                          out audio: Entity(tt:url),
+                          out date: Date,
+    			  out site_name: String,
+    			  out content: String)
+  #_[confirmation="top news on SmartNews"]
+  #_[formatted=[{type="rdl", webCallback="${audio}", displayTitle= "${title}", displayText="ID: ${news_id}\nPublished: ${date}\nSource: ${site_name}"}]]
+    //#_[formatted=[{type="text", text="Id: ${news_id}\nTitle: ${title}\nPublished: ${date}\nSource: ${site_name}\nAudio: ${audio}\nContent: ${content}"}]]
+  #[doc="get the top news from SmartNews"];
```

Comment:
You should have a `#[minimal_projection]` annotation containing the output parameters that should always be included in the result. If you set the entity to be named (and remove the `title` parameter), you should do:
```
#[minimal_projection=['id']]
```
If you set the entity to be unnamed, you should do:
```
#[minimal_projection=['id', 'title']]
```

You should also mark "reading list" as related to this query:
```
#[related=["com.smartnews:reading_list"]]
```